scripts/export-bundles-sizes.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs')
+const path = require('path')
+const { getBrowserSdkVersion } = require('./lib/browser-sdk-version')
+const { getOrg2ApiKey } = require('./lib/secrets')
+const { runMain, fetch: fetchWrapper } = require('./lib/execution-utils')
+
+const rumPath = path.join(__dirname, '../packages/rum/bundle/datadog-rum.js')
+const logsPath = path.join(__dirname, '../packages/logs/bundle/datadog-logs.js')
+const rumSlimPath = path.join(__dirname, '../packages/rum-slim/bundle/datadog-rum-slim.js')
+const workerPath = path.join(__dirname, '../packages/worker/bundle/worker.js')
+
+const LOG_INTAKE_URL = 'https://http-intake.logs.datadoghq.com/api/v2/logs'
+const LOG_INTAKE_REQUEST_HEADERS = {
+  'DD-API-KEY': getOrg2ApiKey(),
+  'Content-Type': 'application/json',
+}
+
+runMain(async () => {
+  const logData = [
+    {
+      message: 'Browser SDK bundles sizes',
+      service: 'browser-sdk',
+      ddsource: 'browser-sdk',
+      env: 'ci',
+      bundle_sizes: {
+        rum: getBundleSize(rumPath),
+        logs: getBundleSize(logsPath),
+        rum_slim: getBundleSize(rumSlimPath),
+        worker: getBundleSize(workerPath),
+      },
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏could it be interesting to monitor all packages instead of only the bundles?